.changeset/polite-queens-thank.md

```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-art': minor
+---
+
+Add PanelSection component and replaces divs with Panel and PanelContent components
+
+BREAKING -> Renamed ConnectionEditor_BooleanEditor to PanelBooleanEditor
```

Comment:
This is of the wrong format, it should be something like

```md
**BREAKING CHANGE:** ...
```

Also, breaking change goes with `major` bump
And lastly, this breaking change should not be here, since you remove it from `legend-application-studio`, your changeset will be over there. Think about what you need to communicate in each module. Here in `@finos/legend-art`, you need to bump `minor` and says: I added this new component, in `legend-application-studio` you say: I'm making a breaking change, I removed this.

Reply:
makes sense!